### PR TITLE
Update data-encoding major version

### PR DESCRIPTION
**Pull request by ia0**

### CODE REVIEW

File: openssl/LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2011 Google Inc.
+Copyright 2011-2017 Google Inc.
```
Comment (sfackler) [inline openssl/LICENSE]: Why?

Comment (ia0) [reply openssl/LICENSE]: As a Google employee, Google claims copyright over my patches [1]. However, it is not explicitly said whether the copyright year matters or not. I can ask on Monday if you want.

[1] https://opensource.google.com/docs/patching/

Comment (sfackler) [reply openssl/LICENSE]: Ah cool, that makes sense :)

**Comment by sfackler** [comment]
Thanks!

**Comment by ia0** [comment]
Thanks for the quick review!

**Pull request merged**